saleor/graphql/shop/mutations/shop_settings_update.py

```diff
@@ -94,6 +101,16 @@ class ShopSettingsInput(graphene.InputObjectType):
             "`shippingPriceCreate` or `shippingPriceUpdate` mutations."
         ),
     )
+    metadata = common_types.NonNullList(
+        MetadataInput,
+        description="Shop public metadata." + ADDED_IN_315,
+        required=False,
+    )
+    private_metadata = common_types.NonNullList(
+        MetadataInput,
+        description=("Shop private metadata." + ADDED_IN_315),
+        required=False,
+    )
```

Comment:
Let's move these declarations above the `#deprecated` comment in line 82. This comment groups the deprecated fields together for better visibility. Metadata fields obviously don't belong to this group.